docs/index.rst

```diff
@@ -459,6 +459,9 @@ The Python bindings also provide some additional configuration properties:
   This callback is served upon calling ``client.poll()`` or ``producer.flush()``. See
   https://github.com/edenhill/librdkafka/wiki/Statistics" for more information.
 
+* ``oauth_cb(oauthbearer_config_str)``: Callback for retrieving Oauth token.
```

Comment:
Add a line saying that oauthbearer_config_str is the value of the `sasl.oauthbearer.config` configuration property.